workflows/example_multifamily_AMY2012.osw

```diff
@@ -356,8 +356,8 @@
         "pg_led": "0",
         "pg_lfl": "0",
         "sch_option_type": "Calculated Lighting Schedule",
-        "weekday_sch": "0.035, 0.033, 0.032, 0.031, 0.032, 0.033, 0.037, 0.042, 0.043, 0.043, 0.043, 0.044, 0.045, 0.045, 0.044, 0.046, 0.048, 0.052, 0.053, 0.05, 0.047, 0.045, 0.04, 0.036",
-        "weekend_sch": "0.035, 0.033, 0.032, 0.031, 0.032, 0.033, 0.037, 0.042, 0.043, 0.043, 0.043, 0.044, 0.045, 0.045, 0.044, 0.046, 0.048, 0.052, 0.053, 0.05, 0.047, 0.045, 0.04, 0.036"
+        "weekday_sch": "0.046, 0.046, 0.046, 0.046, 0.046, 0.037, 0.035, 0.034, 0.033, 0.028, 0.022, 0.015, 0.012, 0.011, 0.011, 0.012, 0.019, 0.037, 0.049, 0.065, 0.091, 0.105, 0.091, 0.063",
+        "weekend_sch": "0.046, 0.046, 0.045, 0.045, 0.046, 0.045, 0.044, 0.041, 0.036, 0.03, 0.024, 0.016, 0.012, 0.011, 0.011, 0.012, 0.019, 0.038, 0.048, 0.06, 0.083, 0.098, 0.085, 0.059"
```

Comment:
@joseph-robertson Did we miss these *.osw update with the lighting schedule Pull request [#419](https://github.com/NREL/OpenStudio-BuildStock/pull/419)?

Reply:
Yes.